list.md

```diff
@@ -7,7 +7,7 @@ librarians.
 Library institutions, journals, and projects
 ---------------------------------------------
 
-SUB Göttingen
+Niedersächsische Staats- und Universitätsbibliothek Göttingen (SUB Göttingen, SUBUGOE)
```

Comment:
Oh, you're right. GitHub has a (rather) new feature that allows that (https://github.com/blog/2141-squash-your-commits). Did'nt know that.